jsonschema-core.xml

```diff
@@ -135,13 +135,67 @@
+                Assertion keywords may be used to describe a set of JSON documents.
+                Without any assertion keywords, the set of all valid instances is the set of all valid JSON documents.
+                Typically, adding assertion keywords reduces the set of valid JSON documents.
+                Keywords are defined to not be redundant with other keywords:
+                For example, the "minimum" keyword only shrinks the valid set of numbers, but not strings or other
+                types&#x2014;this is what the "type" keyword does.
+            </t>
+            <t>
+                When an instance validates against a schema, "annotation keywords" may provide "annotations":
+                metadata that describes the instance.
+                For example, you can document the meaning of a property,
+                suggest a default value for new instances,
+                generate a list of hyperlinks from the instance,
+                or declare relationships between data.
+            </t>
+            <t>
+                "Applicator keywords" are keywords that accept sub-schemas as part of their argument,
+                which are validated against the instance in some fashion.
+                Applicator keywords may use or modify the validation result of their sub-schemas,
+                for example, to be valid conditional on which sub-schemas are valid
+                (e.g. the "if", "then", "oneOf", or "anyOf" keywords).
+                They may also validate a schema against values within the instance,
+                for example, a specific property within the object (the "properties" keyword).
+                The "$ref" keyword is a special applicator keyword that takes a reference to a schema,
+                instead of a literal schema. This allows schemas to be defined recursively.
+            </t>
+            <t>
+                Applicator keywords typically also return the annotations from their valid schemas.
+                Since only annotations from valid schemas can be returned,
+                applicator keywords may be used to specify annotations that conditionally appear,
+                depending on the validation result of other keywords.
+            </t>
+            <t>
+                Since keywords are listed in an object by their name,
+                they may be used at most once per schema and sub-schema.
+                However, the effect of multiple keywords may be achieved by listing each keyword in its own sub-schema,
+                inside an applicator keyword designed for this purpose, such as the "allOf" keyword.
+                Additionally, many keywords can often be reduced to a single keyword;
+                for example, when using "minimum", only the highest value of many will be effective.
+                Schemas may also be a boolean value:
+                The booleans true/false represent a schema that's always valid/invalid, respectively.
```

Comment:
Does this statement deserve its own paragraph?  It seems unrelated to this paragraph.